src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorCollectRunner.cs

```diff
@@ -171,6 +170,9 @@ protected override void StandardOutputCallback(string line)
                 case "Microsoft.Diagnostics.Tools.Monitor.Startup":
                     HandleStartupEvent(logEvent);
                     break;
+                case "Microsoft.Diagnostics.Tools.Monitor.CollectionRules.CollectionRuleService":
```

Comment:
nit: using ifs and typeof(...).FullName might be better here.

Reply:
What do you mean when you say "ifs"? I think of conditionals, as in "if (...) { }", but I don't think that is what you are implying here.